content/efcore/getting-started/samples/loading-related-data-tutorial/min-method/Program.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var year = context.Entry(book)
```

Comment:
Need to load a book before doing this, like in the `Count` example snippet - might be good to load a book with multiple editions like _The Scarlet Plague_